README.md

```diff
@@ -498,3 +498,51 @@ Some of the options include:
 * `members` - automatically document the members in that Python module.
 * `private-members` - include private functions and methods.
 * `show-inheritance` - show the class inheritance.
+
+# Opentelemetry Instrumentation
+
+IIB is now implemented with Opentelemetry python instrumentation.
+This is done in order to collect detailed information about the behavior and performance of IIB.
+By using a combination of automatic and manual instrumentation, it is possible to capture all relevant information about the application's behavior.
+
+## Environment setup
+The two most important packages to be installed are the `opentelemetry-api` and `opentelemetry-sdk`.
+Please see the ``requirements.txt`` file for a more detailed list..
+
+```bash
+pip install virtualenv
+virtualenv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+## Traces
+The application is instrumented to collect traces using OpenTelemetry.
+The traces include information about the duration of each request, as well as any errors that may have occurred.
+
+## Opentelemetry Environment Variables
```

Comment:
I think we should follow the same format in README file where the variables defined as part of configuring each component ( REST API, Worker ) and env ( local env ). When I reviewed the PR, this gave me the impression that I would need to define them once, but looking at the MRs in gitlab, it would be better to define these in https://github.com/release-engineering/iib#configuring-the-workers
https://github.com/release-engineering/iib#configuring-the-rest-api
https://github.com/release-engineering/iib#development-environment

Just like the way that AWS variables are defined.